examples/morpheusvm/scripts/stop.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+# Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+# See the file LICENSE for licensing terms.
+
+set -e
+
+killall avalanche-network-runner
```

Comment:
nit: add new line

Reply:
We should also add this script back as the way to stop the network mentioned in the README